packages/sdk/react-client/src/echo/useQuery.ts

```diff
@@ -8,24 +8,24 @@ import { Space } from '@dxos/client';
 import { Document, DocumentBase, Filter, TypeFilter } from '@dxos/echo-schema';
 
 type UseQuery = {
-  <T extends DocumentBase>(space: Space, filter?: TypeFilter<T>): T[];
-  <T extends DocumentBase>(space: Space, filter?: Filter<T>): Document[];
+  <T extends DocumentBase>(space?: Space, filter?: TypeFilter<T>): T[];
+  <T extends DocumentBase>(space?: Space, filter?: Filter<T>): Document[];
 };
 
 /**
  * Create subscription.
  */
-export const useQuery: UseQuery = <T extends DocumentBase>(space: Space, filter?: Filter<T>): DocumentBase[] => {
+export const useQuery: UseQuery = <T extends DocumentBase>(space?: Space, filter?: Filter<T>): DocumentBase[] => {
```

Comment:
I considered doing this as well but was a little uneasy about returning an empty array when the space was undefined. If you think it's alright then I'm cool with it as it definitely makes the hook easier to work with.

Reply:
we'd do a suspense here if doing this properly right?